### PR TITLE
Long Domain 1um Resolution

### PR DESCRIPTION
**Pull request by ajnonaka**
This is a 4um x 1.6cm x 4um domain with a current source with wavelength 1.6cm.  In the visualizaton, the x and z directions have been stretched by a factor of 100x in order to see the wave (otherwise it was too skinny)
long

### CODE REVIEW

File: Examples/Tests/circuits/inputs_longdomain [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.use_filter = 0
+warpx.cfl = 0.9
+
+algo.em_solver_medium = macroscopic # vacuum/macroscopic
+
+algo.macroscopic_sigma_method = laxwendroff # laxwendroff or backwardeuler
```
Comment (RevathiJambunathan) [inline Examples/Tests/circuits/inputs_longdomain]: this is very minor -- backwardeuler is more stable than laxwendroff and its the default.

File: Examples/Tests/circuits/inputs_longdomain [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+my_constants.pi = 3.14159265359
+my_constants.L = 141.4213562373095e-6
+my_constants.c = 299792458.
+my_constants.wavelength = 1.6e-2
+
+my_constants.sigma_0 = 0.0
+my_constants.epsilon_0 = 8.8541878128e-12
+my_constants.mu_0 = 1.25663706212e-06
+
+my_constants.TP = 2.5e-14
```
Comment (RevathiJambunathan) [inline Examples/Tests/circuits/inputs_longdomain]: is TP used anywhere?

File: Examples/Tests/circuits/inputs_longdomain (hunk 1 above)
Comment (ajnonaka) [reply Examples/Tests/circuits/inputs_longdomain]: sure - sigma is zero so it won't matter either way

Comment (ajnonaka) [reply Examples/Tests/circuits/inputs_longdomain]:
```suggestion
algo.macroscopic_sigma_method = backwardeuler # laxwendroff or backwardeuler
```

File: Examples/Tests/circuits/inputs_longdomain [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+macroscopic.epsilon_function(x,y,z) = "epsilon_0"
+
+macroscopic.mu_function(x,y,z) = "mu_0 "
+
+#################################
+############ FIELDS #############
+#################################
+
+my_constants.pi = 3.14159265359
+my_constants.L = 141.4213562373095e-6
```
Comment (ajnonaka) [inline Examples/Tests/circuits/inputs_longdomain]:
```suggestion
```

File: Examples/Tests/circuits/inputs_longdomain [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+my_constants.c = 299792458.
+my_constants.wavelength = 1.6e-2
+
+my_constants.sigma_0 = 0.0
```
Comment (ajnonaka) [inline Examples/Tests/circuits/inputs_longdomain]:
```suggestion
```

**Review by RevathiJambunathan — APPROVED** [review]
Looks good to me. Thank you Andy

**Pull request merged**